FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice/covg_multiple_priorities_no_timeslice_utest.c

```diff
@@ -631,227 +645,613 @@ void test_task_delay_until_with_config_assert( void )
 }
 
 /**
- * @brief prvAddNewTaskToReadyList - add a newly created task to the list of ready tasks
- *
- * This test creates two tasks, the second after suspending the first and then
- * starts the scheduler.
+ * @brief xTaskPriorityDisinherit - disinherit priority, potentially restoring the original priority used before escalation due to mutex usage
  *
  * <b>Coverage</b>
  * @code{c}
- * if( uxCurrentNumberOfTasks == ( UBaseType_t ) 1 )
+ *     if( pxTCB->uxPriority != pxTCB->uxBasePriority )
+ *     {
+ *     ...
+ *          if( pxTCB->uxMutexesHeld == ( UBaseType_t ) 0 )
  * @endcode
- * As two tasks arecreated, this covers both branches of the above conditional
- * in addition to the function body.
+ *
+ * Cover the case where priority has changed due to inheritance
+ * because of a shared mutex.
+ *
  */
-void test_coverage_prvAddNewTaskToReadyList_create_two_tasks_with_the_first_suspended( void )
+void test_coverage_xTaskPriorityDisinherit_current_is_same( void )
```

Comment:
There are existing test cases for single core.
Why are we cover this function in coverage test?

Reply:
Attached image is the function coverage before this PR.
![image](https://user-images.githubusercontent.com/61685396/223078999-de90a02e-7957-4638-a8c2-98bffb150d88.png)